### ISSUE TITLE
The representation for learnable action tokens.

### ISSUE THREAD

**Issue by wyddmw** [Robot-VLAs/RoboVLMs#20]
Hi, thanks for this awesome work! I would like to ask a simple question why did you choose to use a single learnable token and repeat it multiple times to represent the action instead of using different tokens? Assume we want to represent an action with 4 tokens, what is the difference between tokens = nn.Parameter(torch.zeros(token_num, self.hidden_size)) and tokens = nn.Parameter(torch.zeros(1, self.hidden_size)).repeat(token_num, 1) and is the corresponding advantage?

**Comment by lixinghang12** [comment]
> Hi, thanks for this awesome work! I would like to ask a simple question why did you choose to use a single learnable token and repeat it multiple times to represent the action instead of using different tokens? Assume we want to represent an action with 4 tokens, what is the difference between tokens = nn.Parameter(torch.zeros(token_num, self.hidden_size)) and tokens = nn.Parameter(torch.zeros(1, self.hidden_size)).repeat(token_num, 1) and is the corresponding advantage?

The main difference is that the latter implementation assumes same initial token as input, while the first implementation inputs 4 different tokens. Actually, i think their performance would be similar, welcome the evaluate the first one and update the results.
